### ISSUE TITLE
Organizers block does not permit to show the wordpress.org username and link

### ISSUE THREAD

**Issue by CarlosLongarela** [WordPress/wordcamp.org#1283]
The organizers block does not permit us to show the wordpress. org username and link.

We can set the wordpress.org username for every organizer in the **WordCamp Europe** website on `wcb_organizer` CPT, but this data can be shown on frontend because the organizers block does not have an option to show the username and link to the website.

**To reproduce**

1. Go to wp-admin/edit.php?post_type=wcb_organizer
2. Add or edit a organizer
3. Fill the WORDPRESS.ORG USERNAME on Organizer info
4. Add or modify an organizer page
5. Add or modify an Organizers block
6. There is no option to add the WORDPRESS.ORG USERNAME data

Thanks.

**Comment by ohiawins** [comment]
+1

**Comment by malgrauk** [comment]
I'm sure we logged something similar last November but I cannot locate it. Thanks for adding to the system @CarlosLongarela !

**Comment by patriciabt** [comment]
Indeed, not available in the block (same as with the Speakers block btw)

![image](https://github.com/WordPress/wordcamp.org/assets/1843660/d3f7b619-a74f-4a20-8ab8-8b541d1d1f78)
